ui/raidboss/data/05-shb/dungeon/paglthan.js

```diff
@@ -0,0 +1,107 @@
+import Conditions from '../../../../../resources/conditions';
+import NetRegexes from '../../../../../resources/netregexes';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+
+export default {
+  zoneId: ZoneId.Paglthan,
+  timelineFile: 'paglthan.txt',
+  timelineTriggers: [
+    {
+      // This is a rear cone attack that always follows Wide Blaster.
```

Comment:
Would it make sense to trigger this off of wide blaster, then? (Not that timeline triggers should be avoided, but they feel less safe than triggering off of a log directly.)